streams/src/test/java/org/apache/kafka/streams/processor/internals/PartitionGroupTest.java

```diff
@@ -88,6 +91,14 @@ private static Sensor getValueSensor(final Metrics metrics, final MetricName met
 
     @Test
     public void testTimeTracking() {
+        testFirstBatch();
+        testSecondBatch();
+    }
```

Comment:
The original method was too long and checkstyle failed -- we could also add a checkstyle exception... This was just a quick fix -- let me know what you think

Reply:
Sounds good to me -- better not add more checkstyle exceptions :)